File: HackerRank Solutions/30 Days of Code/Day 1 Data Types/Day 1 Data Types.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";

    
    // Declare second integer, double, and String variables.
    int i1;
    double d1;
    string s1;
    // Read and save an integer, double, and String to your variables.
    // Note: If you have trouble reading the entire string, please go back and review the Tutorial closely.
    cin >> i1;
    cin >> d1;
    cin.get();
    getline(cin, s1);
    // Print the sum of both integer variables on a new line.
    cout << i+i1 << endl;
    // Print the sum of the double variables on a new line.
    d = d+d1;
    cout << std::fixed << std::setprecision(1) << d << endl;
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    cout << s << s1;
    return 0;
}

File: HackerRank Solutions/30Days of Code/Day 1 Data Types/Day 1 Data Types.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {
	
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";
		
        Scanner scan = new Scanner(System.in);

        /* Declare second integer, double, and String variables. */
        int i1 = Integer.parseInt(scan.nextLine());
        double d1 = Double.parseDouble(scan.nextLine());
        String s1 = scan.nextLine();
        /* Read and save an integer, double, and String to your variables.*/
        // Note: If you have trouble reading the entire String, please go back and review the Tutorial closely.
        
        /* Print the sum of both integer variables on a new line. */
        System.out.println(i + i1);
        /* Print the sum of the double variables on a new line. */
		System.out.println(d + d1);
        /* Concatenate and print the String variables on a new line; 
        	the 's' variable above should be printed first. */
        System.out.println(s + s1);
        scan.close();
    }
}